app/utils/helpers.rb

```diff
@@ -13,12 +13,13 @@ def is_time?(t)
   false
 end
 
-def open_now?(place, user_now, place_now)
+def open_now?(opening_hours, user_now, place_now)
   # This method returns true if the user_now (time object) is between the
   # open and close hours of the place. The place_now determines the place's
   # day to match the hours.
+  # opening_hours is a hash of hours of a place
   day = place_now.strftime('%A')[0...3].downcase
-  oh = place[:opening_hours] || {}
+  oh = opening_hours || {}
```

Comment:
we rather not use short variable names (make things obvious and readable rather than short and unreadable for humans)